datashape/py2help.py

```diff
@@ -22,11 +22,12 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-
+import platform
 
 PY2 = sys.version_info[0] == 2
 PY3 = sys.version_info[0] == 3
 
+CPYTHON = platform.python_implementation() == 'CPython'
```

Comment:
I am not running on non-cpython, and while I don't think many people who are doing numerical python are using pypy there is no reason to intentionally break it. DataShape is general enough that I could see it being used on pypy.